pkg/plugins/scorer/load_aware_scorer_test.go

```diff
@@ -72,42 +70,24 @@ func TestLoadBasedScorer(t *testing.T) {
 					},
 				},
 			},
-			wantRes: &types.ProfileRunResult{
-				TargetPods: []types.Pod{&types.ScoredPod{
-					Pod: &types.PodMetrics{
-						Pod: &backend.Pod{
-							NamespacedName: k8stypes.NamespacedName{Name: "pod2"},
-						},
-						MetricsState: &backendmetrics.MetricsState{
-							WaitingQueueSize:    0,
-							KVCacheUsagePercent: 0.2,
-							MaxActiveModels:     2,
-							ActiveModels: map[string]int{
-								"foo": 1,
-								"bar": 1,
-							},
-						},
-					},
-					Score: 0.5,
-				},
-				},
+			want: map[string]float64{
+				"pod1": 0.4,
+				"pod2": 0.5,
+				"pod3": 0,
 			},
 		},
 	}
 
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
-			schedulerProfile := framework.NewSchedulerProfile().
-				WithScorers(framework.NewWeightedScorer(test.scorer, 1)).
-				WithPicker(picker.NewMaxScorePicker(picker.DefaultMaxNumOfEndpoints))
-
-			got, err := schedulerProfile.Run(context.Background(), test.req, nil, test.input)
-
-			if test.err != (err != nil) {
-				t.Errorf("Unexpected error, got %v, want %v", err, test.err)
+			wantScores := map[types.Pod]float64{}
+			for _, pod := range test.input {
+				wantScores[pod] = test.want[pod.GetPod().NamespacedName.Name]
 			}
 
-			if diff := cmp.Diff(test.wantRes, got); diff != "" {
+			got := test.scorer.Score(context.TODO(), nil, nil, test.input)
```

Comment:
can we use context.Background instead of TODO?
```suggestion
			got := test.scorer.Score(context.Background(), nil, nil, test.input)
```